### PR TITLE
Add hmac

### PR DESCRIPTION
**Pull request by tharvik**
Merge both `hmac` version.

Also remove `_operator` which was ~needed by hmac but not anymore (it was useless anyway as it was untyped and also implementation specific).

### CODE REVIEW

File: stdlib/2and3/hmac.pyi [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+from typing import Any, Callable, Optional, Union, overload
+from types import ModuleType
+import sys
+
+_B = Union[bytes, bytearray]
+
+# TODO more precise type for object of hashlib
+_Hash = Any
+
+if sys.version_info >= (3, 4):
```
Comment (gvanrossum) [inline stdlib/2and3/hmac.pyi]: What's the point of the version check? Maybe I'm missing something, but I can't find any difference between the two alternatives below.

Comment (tharvik) [inline stdlib/2and3/hmac.pyi]: Typo, actually, digestmod should not accept a string argument in py < 3.4

**Comment by gvanrossum** [comment]
I guess now you need a `# type: ignore` to silence the mypy checker (I really should prioritize mypy's version checking).

**Comment by tharvik** [comment]
Right, was a bit fast in commiting, should be good now.

**Pull request merged**